src/App.js

```diff
@@ -1,26 +1,28 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
```

Comment:
```suggestion
  const [movies, setMovies] = useState([]);
```
В задании не сказано что нам нужно использовать для стартового отображения данные из файла **movies.json**. Я бы убрал его из стейта.